### DISCUSSION TITLE
Hide title bar for live timing

### DISCUSSION THREAD

**Discussion by DoeEensGek** [f1multiviewer/issue-tracker#377] [Feature requests]
Can you make the title bar hidden for live timing, track map etc.?

I want to use them the same as regular players.
I can't put them at the top of my screen now, and can't put them on top of the taskbar.

If i'm correct:
`const { BrowserWindow } = require('electron')
const win = new BrowserWindow({ width: 800, height: 600, frame: false })
win.show()`

Keeping the window still dragable:
```
<body style="-webkit-app-region: drag">
</body>
```
https://github.com/electron/electron/blob/4e94e0c82f332b668241d8306f7359222c3dd760/docs/api/frameless-window-ko.md#draggable-region

**Comment by 2fst4u** [comment]
Surprising it took me this long to find a request like this. Definitely agree, it's a bit weird that the love timing windows behave differently.

**Comment by krissie-Z** [comment]
Was about to submit my own request on this, I think!

It would be great if the title bars could be optionally turned off for the various live timing windows (eg, sector leaderboard or investigation tracker windows), perhaps allowing their controls to show on hover, or a right click context menu.

This would allow the creation of more seamless setups, with closer integration of data and video windows, without having to bury window title bars beneath videos only to have them show up when clicking the windows.

**Comment by jeppeolesen** [comment]
Marking this as a duplicate of #97
